code/mob/living/critter/gunbot.dm

```diff
@@ -93,3 +94,46 @@
 
 	get_disorient_protection()
 		return max(..(), 80)
+
+	attack_hand(mob/user)
+		user.lastattacked = src
+		if(!user.stat)
+			if (user.a_intent != INTENT_HELP)
+				actions.interrupt(src, INTERRUPT_ATTACKED)
+			switch(user.a_intent)
+				if(INTENT_HELP) //Friend person
+					playsound(src.loc, 'sound/impact_sounds/Generic_Shove_1.ogg', 50, 1, -2)
+					user.visible_message("<span class='notice'>[user] gives [src] a [pick_string("descriptors.txt", "borg_pat")] pat on the [pick("back", "head", "shoulder")].</span>")
+				if(INTENT_DISARM) //Shove
+					SPAWN_DBG(0) playsound(src.loc, 'sound/impact_sounds/Generic_Swing_1.ogg', 40, 1)
+					user.visible_message("<span class='alert'><B>[user] shoves [src]! [prob(40) ? pick_string("descriptors.txt", "jerks") : null]</B></span>")
+				if(INTENT_GRAB) //Shake
+					if (istype(user, /mob/living/carbon/human/machoman))
+						. = ..()
+						return
+					playsound(src.loc, 'sound/impact_sounds/Generic_Shove_1.ogg', 30, 1, -2)
+					user.visible_message("<span class='alert'>[user] shakes [src] [pick_string("descriptors.txt", "borg_shake")]!</span>")
+				if(INTENT_HARM) //Dumbo
+					if (istype(user, /mob/living/carbon/human/machoman))
+						. = ..()
+						return
+					if (ishuman(user) && !ishunter(user) && !iswerewolf(user) && !iswrestler(user))
```

Comment:
I'm not sure I get the werewolf, hunter and wrestler checks. Do those do their own thing in their hand_attack procs? If so maybe this should have a comment about that.

Reply:
Werewolves have a custom attack for regular cyborgs and can smash their heads in like two attacks. This lets gunbots be attacked by barehand WW, although it uses regular claw attacks and the WW doesn't kill them anywhere near as easily.

Wrestlers and hunters don't actually get the ability to punch regular cyborgs so maybe that should be removed.

Reply:
🤷‍♀️